migrations/src/postDeploymentSetup.js

```diff
@@ -36,7 +34,7 @@ module.exports = async () => {
   const { contractInstance: rewardManager } = await getdeployedContractInstance('RewardManager', rewardManagerAddress);
   const { contractInstance: voteManager } = await getdeployedContractInstance('VoteManager', voteManagerAddress);
   const { contractInstance: delegator } = await getdeployedContractInstance('Delegator', delegatorAddress);
-  const { contractInstance: schellingCoin } = await getdeployedContractInstance('SchellingCoin', schellingCoinAddress);
+  const { contractInstance: RAZOR } = await getdeployedContractInstance('RAZOR', RAZORAddress);
```

Comment:
Though very small change
Please keep casing consistent
somewhere its razor, somewhere its Razor (tests), somewhere its RAZOR(contract instance)

Reply:
yeah I have used following naming convention
SchellingCoin -> RAZOR
schellingCoin -> Razor
sch -> razor

Reply:
@adi44 
Again see we are debating on naming convention :joy:
Productive it is :new_moon:
IMO
shellingCoin was making sense as it was camelcase 
Can we have RAZOR as contract name and razor wherever we are using instance 
Take action as per your understanding

Reply:
+